client/app/locales/en.json

```diff
@@ -40,6 +40,7 @@
     "login welcome": "Welcome %{username}",
     "login enter your password": "Please enter your password to access to your Cozy",
     "login password": "password",
+    "login hotp": "Authentication code",
```

Comment:
I think the key is now `login otp` (see https://github.com/cozy/cozy-proxy/pull/260/files#diff-ba31741f95a1b01c92076116ff07bd93R11)